integreat_cms/static/src/js/mutually_exclusive_checkboxes.ts

```diff
@@ -0,0 +1,23 @@
+const enforceMutualExclusion = (clickedCheckbox: HTMLInputElement) => {
+    // ignore disabled checkboxes and allow un-checking of all boxes
+    if (clickedCheckbox.disabled || !clickedCheckbox.checked) {
+        return;
+    }
+    const checkboxes = <HTMLInputElement[]>(<any>document.querySelectorAll(".mutually-exclusive-checkbox"));
+    checkboxes.forEach((checkbox) => {
+        if (!checkbox.disabled) {
+            // eslint-disable-next-line no-param-reassign
+            checkbox.checked = false;
+        }
+    });
+    // eslint-disable-next-line no-param-reassign
+    clickedCheckbox.checked = true;
+};
+
+window.addEventListener("load", () => {
+    const checkboxes = <HTMLInputElement[]>(<any>document.querySelectorAll(".mutually-exclusive-checkbox"));
+    checkboxes.forEach((checkbox) => {
+        // eslint-disable-next-line no-param-reassign
+        checkbox.addEventListener("click", () => enforceMutualExclusion(checkbox));
```

Comment:
Oh wow, no idea how I missed that! That's a bit embarassing.

Hm, the only other way I see is an additional event listener for the `automatic_translation` checkbox which adds the `hidden` class back to the `language-options` on uncheck. It might be a bit less "hacky", but not sure if it's worth the effort, since currently the mutual exclusion is not used anywhere else.

Your call, if you are reasonably certain we won't be needing to reuse this anywhere else, I'll go with your original suggestion.

Reply:
I'm not sure I understand, just setting `checkbox.checked = false;` somewhere doesn't fire any events, not even change events, right?
Also, my suggestion should be reusable for other use cases as well, or am I missing something?

Reply:
Just tested it, yep you're right, that doesn't fire the event. Hm. The only thing I'm taking an issue with is removing and readding the event listener.

I just pushed an alternative suggestion, does this work for you?

Reply:
Ah yeah, dispatching the change event makes even more sense to avoid having to unregister the listener :+1:
Then we just have to make sure we remember this when we want to use the click event on mutually exclusive fields in the future :grinning: